.travis.yml

```diff
@@ -1,6 +1,11 @@
-sudo: required # required to install softhsm for PKCS 11 tests
+sudo: false
```

Comment:
Is this the same as not declaring "sudo" at all? Could we just remove the line?

Reply:
But then we couldn't relish the glory of defeating the `sudo` monster each time we looked inside this  file...